cpp/src/parquet/benchmark_util.h

```diff
@@ -0,0 +1,83 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+#include <random>
+#include <string>
+
+#include "parquet/encoding.h"
+
+namespace parquet::benchmark {
+
+void GenerateRandomString(uint32_t length, uint32_t seed, std::vector<uint8_t>* heap) {
+  // Character set used to generate random string
+  const std::string charset =
+      "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+
+  std::default_random_engine gen(seed);
+  std::uniform_int_distribution<uint32_t> dist(0, static_cast<int>(charset.size() - 1));
+
+  for (uint32_t i = 0; i < length; i++) {
+    heap->push_back(charset[dist(gen)]);
+  }
+}
+
+constexpr uint32_t kDefaultDataStringLength = 8;
+
+template <typename T>
+void GenerateBenchmarkData(uint32_t size, uint32_t seed, T* data,
```

Comment:
Same here.

Reply:
Hmmm it's a template function, should I declare different types here, or just let it be a inline template in `.h`?

Reply:
Can use https://en.cppreference.com/w/cpp/language/function_template#Function_template_instantiation

Reply:
done it, mind take a look?

Reply:
Seems ok to me, thanks!